domain/users/users_service.go

```diff
@@ -271,21 +301,28 @@ func (s *UserService) GetUserBalance(accessKey string) (*Balance, error) {
 	// Create BUX client with access key.
 	buxClient, err := s.buxClientFactory.CreateWithAccessKey(accessKey)
 	if err != nil {
-		s.log.Error(err.Error())
+		s.log.Error().
+			Str("accessKey", accessKey).
```

Comment:
Is it okay to print access key?

Reply:
You are right, it can be a security issue. I removed it